### PR TITLE
Add the PROG as-of-now

### PR DESCRIPTION
**Pull request by albertmink**
see #92

### CODE REVIEW

**Comment by albertmink** [comment]
I tested the XSLT and JSON Schema generation with this program for `if_aff_intf_v1`. This works, but I am unfamiliar with the other features.
So some proper testing would be nice.

If the report does what is should do, I would like to merge into main. And then tackle the broken unit tests, ok?

**Review by wurzka — CHANGES_REQUESTED** [review]
I made a short test of the report in our system. Can you also add the text elements from the origin report to this one?

There is also an issue with printing the log, but this can be fixed in an other PR.

**Comment by albertmink** [comment]
I opened an issues for the brocken log see #99

File: src/z_generate_repo.prog.xml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    <item>
+     <ID>S</ID>
+     <KEY>P_TYPE</KEY>
+     <ENTRY>Type Name</ENTRY>
+     <LENGTH>17</LENGTH>
+    </item>
+    <item>
+     <ID>S</ID>
+     <KEY>P_XSLT</KEY>
+     <ENTRY>Generate Simple Transformation</ENTRY>
```
Comment (wurzka) [inline src/z_generate_repo.prog.xml:59]: There shall be also texts for p_consol, p_disk, p_examp,...

**Review by wurzka — CHANGES_REQUESTED** [review]
Sorry @albertmink , I think I have expressed myself in a misleading way.
With 'Text Elements', I meant the Selection Texts for the UI Elements of the report

File: src/z_generate_repo.prog.xml (hunk 1 above)
Comment (albertmink) [reply src/z_generate_repo.prog.xml:59]: ahh, the report was simply duplicated in ADT. So this is strange 🤕  Will try do fix it for the report here

**Review by wurzka — APPROVED** [review]
Thanks @albertmink, looks good to me :)

**Pull request merged**